src/components/home/hero_section/title/animated_letters.js

```diff
@@ -0,0 +1,89 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { motion } from "framer-motion"
+
+import Planet from "../../../planet"
+import useDetectJavascript from "../../../../utils/use_detect_javascript"
+
+import styles from "./animated_letters.module.scss"
+
+const dragVariants = {
+  in: {
+    opacity: 1,
+    transition: {
+      ease: "easeIn",
+      delay: 1.5,
+      delayChildren: 1.5,
+      staggerChildren: 0.04,
+    },
+  },
+  out: {
+    opacity: 0,
+  },
+}
+
+const letterVariants = {
+  in: {
+    opacity: 1,
+  },
+  out: {
+    opacity: 0,
+  },
+}
+
+const renderAnimatedLetters = string =>
+  Array.from(string).map((letter, index) => (
+    // eslint-disable-next-line react/no-array-index-key
```

Comment:
I disable this all together always. It's a rule that is rarely used and only usefull when dealing with animations and dynamic lists.

Reply:
This is the only instance of this warning in the codebase. I can disable it completely but I don't think it is worth right now to be honest

Reply:
Let it be then